scripts/raceday_verify.py

```
@@ -0,0 +1,848 @@
+    print(f"  Race Day Verification  [{now_str}]")
+    print(f"  Phase: {args.phase:12s}  Date: {race_date}")
+    print(f"  DB:    {Path(args.db).resolve()}")
+    print(f"{'='*65}\n")
+
+    # DB connection
+    if not Path(args.db).exists():
+        if args.phase not in ("quickstart", "all"):
+            print(f"[ERROR] DB not found: {Path(args.db).resolve()}")
+            print("  Run: python -m src.cli.main create-tables")
+            sys.exit(2)
+        print(f"[INFO] DB not found (OK for quickstart phase)")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)